lib/application.js

```diff
@@ -383,6 +384,50 @@ app.set = function set(setting, val) {
   return this;
 };
 
+/**
+ * Extend `setting` with `val`, or return `setting`'s value.
+ *
+ *    app.set('foo', { 'bar' : 1 })
+ *    app.expand('foo', { 'baz' : 1 });
+ *    app.expand('foo');
```

Comment:
@wesleytodd thoughts on this? I'm not sure what the use of having `.extend` called with one argument returning the current value -- that's already covered with `.get` and `.set` I don't think we need a third method to get the value of a setting, right?

Reply:
Agreed.  Not sure if it should error, but it shouldn't do anything and shouldn't be documented.

Reply:
May be useful to throw error that required argument is missing.

Reply:
OK, I will remove this later today.